packages/react-components/src/types/ChatMessage.ts

```diff
@@ -119,6 +123,31 @@ export interface ContentSystemMessage extends SystemMessageCommon {
   content: string;
 }
 
+/* @conditional-compile-remove(data-loss-prevention) */
+/**
+ * Content blocked message type.
+ *
+ * Content blocked messages will rendered default value, but applications can provide custom strings and icon to renderers.
+ *
+ * @beta
+ */
+export interface BlockedMessage extends MessageCommon {
+  messageType: 'blocked';
+
```

Comment:
Remove empty line here

Reply:
was following ChatMessage, but will also remove the line there